RELEASING.md

```diff
@@ -0,0 +1,20 @@
+# Releasing
+
+When releasing, ensure you have write access to the `azavea` 
+bintray organization. You can configure your credentials by
+following the prompts after running `./sbt bintrayChangeCredentials`
+
+## Publishing a release
+
+1. Create a new release branch from up-to-date master, e.g. `release/x.y.z`
+1. Bump version refs in README
+1. Bump version ref in `project/Version.scala`
+1. Review CHANGELOG.md. Move `[Unreleased]` header to empty section and replace with `[x.y.z]` header plus release date. 
+1. Commit these changes as a single commit, with the message "Release vx.y.z"
+1. Push branch and make a PR on GitHub
+1. Ensure CI succeeds
+1. Tag the "Release vx.y.z" commit: `git tag -a vx.y.z -m "Release x.y.z"`
+1. Publish the tagged commit as a new release: `./sbt publish`
+1. Merge release branch
```

Comment:
What happens if another PR is merged (or commits pushed to `master`) before the release PR is merged?

Reply:
We take gun, we shoot guy who merge.</russian>

Reply:
Is it the right thing to move the actual merge up one or two spots?  The publish can happen later with no ill effects, and will shrink the window for bad merges/commits.  Although, for an organization that has few committers, I wonder how risky this is.

Reply:
Yeah. That's fine, I think that's how I originally had it, and is how I did the RC3 merge. I moved the merge item up to just after "Ensure CI succeeds". I also added a note to rebase on master and go back to step two if there are new commits on master.